File: src/main/java/ru/spb/sspk/ssdmd/phonebook_test/service/UserService.java
package ru.spb.sspk.ssdmd.phonebook_test.service;

import org.springframework.stereotype.Service;
import ru.spb.sspk.ssdmd.phonebook_test.model.dto.UserDto;

@Service
public interface UserService {

    UserDto save(Long userId);

    String findAll(Long userId);
}

File: src/main/java/ru/spb/sspk/ssdmd/phonebook_test/service/UserService.kt
package ru.spb.sspk.ssdmd.phonebook_test.service

interface UserService {

    fun create(userId: Long, username: String): String

    fun checkingUserPresence(userId: Long): Boolean

    fun checkingForAuthenticationNow(userId: Long): Boolean

    fun checkingForAuthenticationAndAccess(userId: Long, username: String, answer: String?): String

    fun findById(userId: Long): String

    fun checkingUserRole(userId: Long): Boolean

    fun updateActivityAtToUser(userId: Long)

    fun findAllUsers(): String
}